test/Maestro.ScenarioTests/TestParameters.cs

```diff
@@ -19,21 +20,49 @@ public class TestParameters : IDisposable
     {
         internal readonly TemporaryDirectory _dir;
 
-        public static async Task<TestParameters> GetAsync()
+        private static readonly string[] maestroBaseUris;
+        private static int maestroBaseUriIndex = 0;
+        private static readonly string maestroToken;
+        private static readonly string githubToken;
+        private static readonly string darcPackageSource;
+        private static readonly string azdoToken;
+        private static readonly SemaphoreSlim mutex;
+
+        static TestParameters()
         {
             IConfiguration userSecrets = new ConfigurationBuilder()
                 .AddUserSecrets<TestParameters>()
                 .Build();
 
-            string maestroBaseUri = Environment.GetEnvironmentVariable("MAESTRO_BASEURI") ??  userSecrets["MAESTRO_BASEURI"] ?? "https://maestro-int.westus2.cloudapp.azure.com";
-            string maestroToken = Environment.GetEnvironmentVariable("MAESTRO_TOKEN") ?? userSecrets["MAESTRO_TOKEN"];
-            string githubToken = Environment.GetEnvironmentVariable("GITHUB_TOKEN") ?? userSecrets["GITHUB_TOKEN"];
-            string darcPackageSource = Environment.GetEnvironmentVariable("DARC_PACKAGE_SOURCE");
-            string azdoToken = Environment.GetEnvironmentVariable("AZDO_TOKEN") ?? userSecrets["AZDO_TOKEN"];
+            maestroBaseUris = (Environment.GetEnvironmentVariable("MAESTRO_BASEURIS")
+                    ?? userSecrets["MAESTRO_BASEURIS"]
+                    ?? "https://maestro-int.westus2.cloudapp.azure.com")
```

Comment:
Should this be updated to the new one?

Reply:
Probably, it's not really used but I think it's correct

Reply:
It's used when you don't override this via an env possibly?

But why not `https://maestro.int-dot.net`?

Reply:
didn't push, sorry